source/export-to-language.txt

```diff
@@ -25,6 +26,11 @@ You can export queries and pipelines to the following languages:
 - Ruby
 - Rust
 
+.. note::
+
```

Comment:
Thoughts on moving this under pre-req or does it not make sense?  We can combine it with the note in the pre-req section so it would read like:

```
The :guilabel:`Export to Language` feature is only available if you enable |copilot|. You can also open a new playground to test and export queries using the Copilot extension. For more information, see [Export and Test Queries with Copilot.](https://deploy-preview-104--docs-mongodb-vscode.netlify.app/copilot-export-test/#std-label-vsce-test-queries-copilot)`

Reply:
I moved the info about requiring copilot under prerequisites! However, I'm inclined to keep this info separate from the existing note because the first point is a necessary detail for the Export to Language feature and the existing note is optional additional information.